lib/services/cups.pm

```diff
@@ -29,6 +29,8 @@ sub config_service {
         script_run 'echo FileDevice Yes >> /etc/cups/cupsd.conf';
         assert_script_run 'cupsd';
     }
+
+    record_info('cups-files', script_output('cat /etc/cups/cups-files.conf'));
```

Comment:
Does this file matter (or even exist) in the non-systemd case?

Reply:
Oh, I have assumed they are the same. Fixed, thanks!